doc/contributing/distribution.md

```diff
@@ -25,3 +25,11 @@ only as a semver-major change, unless the related feature or project is
 documented as experimental. In addition, Node.js includes external projects as
 internal dependencies. These may be replaced or removed at any time, provided
 that doing so is not a breaking change.
+
+## Executables
+
+The Node.js distribution adds a number of executable commands into the user's
```

Comment:
> The Node.js distribution adds a number of executable commands into the user's `PATH`

Are you talking about the tarball? The tarball doesn't add anything to PATH. The installers might (optional, I guess). But the majority of our users don't use the installers - they either use the version manager or some system package manager. The clearer way to describe what actually happens should be something like

```
The Node.js distribution includes a few executables. Those that are meant to be added to user's
`PATH` environment variable are placed under the `bin/` directory in the distribution, such as `node`
This directory will only include executables that run code that is vendored within the Node.js
distribution. Executables whose names are in conflict with well-known software that's not vendored
within Node.js itself will not be added to the `bin/` directory.
```

Reply:
Yeah it’s tricky to describe this both correctly and succinctly. Perhaps “The Node.js includes a few executables, such as `node`.” is enough, without the discussion of where they’re saved?